semantic.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
you can remove this file, with the mix we don't need the **gulp** to build the script